molgenis-jobs/src/main/java/org/molgenis/data/jobs/model/JobExecution.java

```diff
@@ -197,7 +197,7 @@ public void setFailureEmail(String failureEmail)
 		PENDING, RUNNING, SUCCESS, FAILED, CANCELED
 	}
 
-	private void setDefaultValues()
+	public void setDefaultValues()
 	{
 		setSubmissionDate(Instant.now());
```

Comment:
setAuto(true) in metadata would inject the current date so we can get rid of the setDefaultValues